Makefile

```
@@ -8,7 +8,7 @@ cmd/ecspresso/ecspresso: *.go cmd/ecspresso/*.go go.* appspec/*.go
 	cd cmd/ecspresso && go build -ldflags "-s -w -X main.Version=${GIT_VER} -X main.buildDate=${DATE}" -gcflags="-trimpath=${PWD}"
 
 install: cmd/ecspresso/ecspresso
-	install cmd/ecspresso/ecspresso ${GOPATH}/bin
+	install cmd/ecspresso/ecspresso ${GOPATH}/ecspresso
```

```suggestion
	install cmd/ecspresso/ecspresso `go env GOPATH`/bin/ecspresso
```

`make install` intends to install a binary to the same path of `go get`.
Go 1.8 or later enables to get the path by `go env GOPATH`. I should use that value.

---

Thank you for reply!
I made a change along with your suggestion. 😄

---

Check behavior is blow.
```
ecspresso on feature/add_make_install via 🐹 v1.15.2 took 3s 
❯ make install
cd cmd/ecspresso && go build -ldflags "-s -w -X main.Version=v1.1.2-1-gff361f1 -X main.buildDate=2021-02-16T00:03:07+0900" -gcflags="-trimpath=/Users/naoto/ghq/github.com/fnaoto/ecspresso"
install cmd/ecspresso/ecspresso `go env GOPATH`/bin/ecspresso
ecspresso on feature/add_make_install [*1] via 🐹 v1.15.2 took 5s 

❯ ls `go env GOPATH`/bin/ecspresso
/Users/naoto/go/bin/ecspresso

ecspresso on feature/add_make_install [*1] via 🐹 v1.15.2 
❯ type ecspresso
ecspresso is /Users/naoto/go/bin/ecspresso

ecspresso on feature/add_make_install [*1] via 🐹 v1.15.2 
❯ ecspresso --help
usage: ecspresso [<flags>] <command> [<args> ...]

```